cmake/CPM.cmake

```diff
@@ -505,3 +509,62 @@ function(cpm_is_git_tag_commit_hash GIT_TAG RESULT)
     endif()
   endif()
 endfunction()
+
+macro(prettify_cpm_add_package OUT_VAR IS_IN_COMMENT)
```

Comment:
Could you change the name to `cpm_prettyfy_package_arguments`?
That way it is more clear and prefixed with `cpm`.